docs/design.html

```diff
@@ -523,9 +523,13 @@ <h4><a id="design_compactiondetails" href="#design_compactiondetails">Log Compac
     <h4><a id="design_compactionconfig" href="#design_compactionconfig">Configuring The Log Cleaner</a></h4>
 
     The log cleaner is enabled by default. This will start the pool of cleaner threads.
-    To enable log cleaning on a particular topic you can add the log-specific property
-    <pre class="brush: text;">  log.cleanup.policy=compact</pre>
-    This can be done either at topic creation time or using the alter topic command.
+    To enable log cleaning on a particular topic, add the log-specific property
+    <pre class="brush: text;">log.cleanup.policy=compact</pre>.
```

Comment:
remove the period after `</pre>` (`<pre>` puts the string on line by itself); or change to `<code>` (`<code>` puts string inline).